db.go

```diff
@@ -513,6 +526,7 @@ func NewDB(db *sql.DB, options ...DBOption) (*DB, error) {
 		IgnoreMissingCols:     false,
 		Logger:                nil,
 		OpentracingEnabled:    false,
+		SpanOptions:           make([]opentracing.StartSpanOption, 0),
```

Comment:
Could probably initialize to `nil` just fine

Reply:
updated